admin-dev/themes/new-theme/js/app/utils/init-components.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+import TranslatableInput from '@js/components/translatable-input.js';
+
+const initComponents = () => {
+  // Initialize the prestashop global object
+  window.prestashop = {
+    components: {
+      enableTranslatableInput,
+    },
+  };
+};
+
+export default initComponents;
+
+const enableTranslatableInput = function () {
+  if (window.prestashop.components.enableTranslatableInput.enabled) {
+    return;
+  }
+  new TranslatableInput();
+  window.prestashop.components.enableTranslatableInput.enabled = true;
+};
```

Comment:
```suggestion
const initComponents = () => {
  // Initialize the prestashop global object
  window.prestashop = {
    components: {
      translatableInput: {
        enabled: false,
        enable() {
          // Already enabled
          if (window.prestashop.translatableInput.enabled === true) {
            return;
          }

          new TranslatableInput();
          window.prestashop.translatableInput.enabled = true;
        },
      }
      enableTranslatableInput,
    },
  };
};

export default initComponents;
```

Not sure your code will work if you call `window.prestashop.components.enableTranslatableInput()` twice, you change the object definition when calling this method

Reply:
You also forget to return the instance of TranslatableInput component :thinking:

Reply:
Your suggestion looks great, but I don't quite understand one thing: why "enableTranslatableInput," is still there if the function was deleted and enabling works diffrently now?

Reply:
It's for the eslint, you can't have anonymous function without name :sweat_smile:

Reply:
It was a residue from the git change :sweat_smile: 
The correct implementation: https://github.com/PrestaShop/PrestaShop/pull/20591#discussion_r475110356